_includes/data-table.html

```diff
@@ -0,0 +1,24 @@
+<div class="figure">
+  <h1 class="figure__title">New America Open COVID Project List</h1>
+
+  <table class="data-table">
+    <thead>
+      <tr>
+        <th scope="col">Project title</th>
+        <th scope="col">Description</th>
+        <th scope="col">Source</th>
+      </tr>
+    </thead>
+    <tbody>
+      {% for project in site.data.projects %}
+      <tr>
+        <th scope="row"><a href="{{ site.baseurl }}/projects/{{ project.title | slugify: "latin" }}">{{project.title}}</a></th>
+        <td>{{project.summary}}</td>
+        <td>{{project.organizing-org | default: "N/A"}}</td>
+      </tr>
+      {% endfor %}
+    </tbody>
```

Comment:
Is this code the same as what's in index.html? If so, maybe we should extract it further into another _includes. That way, if I make a change (say, from Source to Type of Product, which I might end up doing), we can do it in one place instead of multiple?

Reply:
Yes! I was doing the minimum to make this workable, but would recommend using an include.

Reply:
Alright, made the change and it's now included in index.html too!